charts/peerscout/values.yaml

```diff
@@ -42,7 +42,9 @@ service:
 
 ingress:
   enabled: false
-  annotations: {}
+  annotations:
+    nginx.ingress.kubernetes.io/proxy-connect-timeout: "90"
```

Comment:
Thank you! There is a quick [explanation](https://loft.sh/blog/kubernetes-nginx-ingress-10-useful-configuration-options/#timeout-settings) for it in case of you want to look at it.